### ISSUE TITLE
[Feature] GOG Version Support

### ISSUE THREAD

**Issue by InsanePanda** [vs-shirokii/gzdoom-rt#1]
Running the .exe gives a dialogue box that says it cannot locate the wad files and to install via Steam.

But the game is also available on GOG so it should not have a Steam requirement.

https://www.gog.com/en/game/doom_doom_ii

**Comment by brainhaver** [comment]
> Basically install "DOOM + DOOM II" on Steam, and the Doom 2: Ray Traced launcher will find it.
>
>If it can't, place "doom2.wad" from DOS version of Doom II into the same folder as gzdoom.exe

**Comment by xtcrefugee** [comment]
Yeah, I'm not sure why it makes it sound like the Steam version specifically is required. The doom2.wad from other retail releases seems to work, just like with normal GZDoom.

I've successfully used the one from the standard retail release (v1.9) which has a SHA-1 hash of 7ec7652fcfce8ddc6e801839291f0e28ef1d5ae7

**Comment by tweety3d** [comment]
Doesn't work with DOOM + DOOM2 version from GOG, but works with just DOOM 2 version from GOG.

**Comment by Hexaae** [comment]
Yep, original DOOM.WAD doesn't work. It seems it requires Steam customized DOOM.WAD only. 😫

EDIT: I got the Steam DOOM.WAD but still doesn't work (greyed out). It's probably not supported yet either way....

_Interesting facts:_
Steam version (DOOM1+2) DOOM.WAD is "DOOM: Unity Edition" 12MB
GOG/Original DOOM.WAD = "DOOM Registered" 10MB

**Comment by kolomeets80** [comment]
There's a check that accepts wad files with certain sizes only. This checks needs to be removed or relaxed/made optional.

**Comment by Hexaae** [comment]
I think there's a misunderstanding: I don't think this port supports DOOM 1 at all, at the moment. The label on the download page is clear: "Doom II: Ray Traced".
And as I said I also tried with DOOM.WAD from Steam Doom1+2 and it doesn't work.

**Comment by rcerrejon** [comment]
What about the PC Gamepass version?
Thanks.

**Comment by xproflipscarab** [comment]
Anyone figure this out?
I've tried dragging doom2.wad directly onto gzdoom.exe but still throws the error...
This should not be an issue whatsoever on a DRM free game... why gatekeep a mod that needs a single DRM free game file to run

**Comment by vs-shirokii** [comment]
Sry, I don't "gate-keep it", but every doom2.wad is different, and it was quite a pain to figure out that one doom2.wad has a specific wall slightly different / another .wad has floor texture that isn't load / etc. So the work that is needed to support all doom2.wad-s is immense 🍞

Steam just has the original DOS doom2.wad. 

Either way, one may force-start a custom doom2.wad with:

`gzdoom.exe -rtnolauncher`

This will show semi-original GZDoom launcher, from where one can start any base IWAD.
And to load PWAD, drag-and-drop or pass it as an additional argument to gzdoom.exe

**Comment by JulienR0y** [comment]
Would supporting GOG make supporting the gamepass version possible ?

**Comment by Hexaae** [comment]
> Sry, I don't "gate-keep it", but every doom2.wad is different, and it was quite a pain to figure out that one doom2.wad has a specific wall slightly different / another .wad has floor texture that isn't load / etc. So the work that is needed to support all doom2.wad-s is immense 🍞
> 
> Steam just has the original DOS doom2.wad.
> 
> Either way, one may force-start a custom doom2.wad with:
> 
> `gzdoom.exe -rtnolauncher`
> 
> This will show semi-original GZDoom launcher, from where one can start any base IWAD. And to load PWAD, drag-and-drop or pass it as an additional argument to gzdoom.exe

An option to lunch Doom2 directly? No useless launch panel since only D2 works?

**Comment by rcerrejon** [comment]
I just tested it with the command: `gzdoom.exe -rtnolauncher` with the .wad of Doom 2 from Doom+Doom II (PC Game Pass version), it works great, like a charm.
Doom 1 does not work.
